ui/src/js/components/labbook/overview/FilePreview.scss

```
@@ -6,8 +6,72 @@
 
 .FilePreview__container {
   align-items: center;
-  @include flex(space-between, row);
+  @include flex(flex-start, row);
+  color: $jet;
   a {
-    color: $dark-blue;
+    color: $azure;
+    text-decoration: none;
+    margin-left: 20px;
+    margin-top: 40px;
   }
 }
+
+.FilePreview__empty{
+  width: 100%;
+  height: 100px;
+  background: $grey-border;
+  @include flex(center, row);
+  text-align: center;
+  padding: 30px;
+  position: relative;
+  margin: 0 12.5px;
+
+  button{
+    transition: all 0.25s ease, background-position 1ms;
+    width: 30px;
+    height: 30px;
+    border-radius: 30px;
+    position: absolute;
+    right: 15px;
+    top: -20px;
+    padding: 0;
+    background: $azure url('~Images/icons/continue-arrow.svg') no-repeat -1px center;
+    background-size: 30px;
+    border: none;
+    span {
+      display: none;
+    }
+
+    &:hover,
+    &:focus {
+      border: none;
+      width: auto;
+      padding: 0 40px 0 15px;
+      background: $azure url('~Images/icons/continue-arrow.svg') no-repeat right 1px center;
+      background-size: 30px;
+
+      span {
+        display: block;
+      }
+    }
+  }
+
+}
+
+
+.FilePreview__empty-content{
+  @include flex(center, column);
+  p {
+    margin: 0;
+  }
+  p:first-child {
```

psuedo selectrors should only be used when neccesary, otherwise use BEM

```suggestion
  .FilePreview__paragraph--large{
  font-weight: 900;
  }
  
    .FilePreview__paragraph--darkGrey{
  color: $darkGrey;
  }
```